tools/tpm2_loadexternal.c

```diff
@@ -121,7 +121,7 @@ static bool on_option(char key, char *value) {
         ctx.flags.H = 1;
     break;
     case 'u':
-        if(!files_load_public(optarg, &ctx.public_key)) {
```

Comment:
Indeed, these are left overs from when the tools were ported to the new option handling infrastructure.